docs/sources/administration/roles-and-permissions/access-control/troubleshooting/index.md

```diff
@@ -0,0 +1,91 @@
+---
+aliases:
+  - ../../../enterprise/access-control/troubleshooting/
+cascade:
```

Comment:
Note that there seem to be some changes in cascading - see [this PR](https://github.com/grafana/writers-toolkit/pull/234/files) updating the docs. We might need to add the target path, as far as I understand to have this working?